runelite-api/src/main/java/net/runelite/api/ObjectID.java

```diff
@@ -505,7 +505,7 @@
 	public static final int NULL_499 = 499;
 	public static final int NULL_500 = 500;
 	public static final int NULL_501 = 501;
-	public static final int NULL_502 = 502;
+	public static final int SNOW_ATTACK_CENTER_ID = 502;
```

Comment:
This file is auto-generated. Do not rename fields in this class as it will break upon redumping the objects.